pkg/kubelet/certificate/certificate_store.go

```diff
@@ -307,3 +313,22 @@ func loadX509KeyPair(certFile, keyFile string) (*tls.Certificate, error) {
 	cert.Leaf = certs[0]
 	return &cert, nil
 }
+
+// remove obsolete certificates belong to self, but reserve current one.
```

Comment:
What about "removeObsolete removes old versions of rotated file"?

Reply:
outdated certificates. kubelet-server only delete server's certs, client delete client's certs.